compose.c

```diff
@@ -2104,13 +2139,15 @@ int mutt_compose_menu(struct Email *e, struct Buffer *fcc, struct Email *e_cur,
           mutt_str_copy(buf, CUR_ATTACH->body->language, sizeof(buf));
         if (mutt_get_field("Content-Language: ", buf, sizeof(buf), MUTT_COMP_NO_FLAGS) == 0)
         {
+          text_field_updated = strcmp(CUR_ATTACH->body->language, buf) != 0;
           CUR_ATTACH->body->language = mutt_str_dup(buf);
           menu->redraw |= REDRAW_CURRENT | REDRAW_STATUS;
           mutt_clear_error();
```

Comment:
@flatcap I'm wondering what might've generated an error that needs to be cleared here? Within the scope of the case there are calls to `mutt_str_copy` and `mutt_get_field` (though this block is only if that returns `0`).

I ask because other `case`s are doing similar things but without a call to `mutt_clear_error` so I'm wondering if maybe they should also be clearing or if this can be removed. (or maybe there's just something in the broader scope I'm missing)

Reply:
`mutt_clear_error()` is an artifact of some old code -- they _shouldn't_ be needed.
Leave it in, for now.  I've made myself a note to investigate, another time.

---

Historically, the code used to write to absolute positions on screen.
After some input code, it was common to see the caller try to clear up afterwards.